compiler/syntax/src/jsx_v4.ml

```diff
@@ -1002,7 +1002,8 @@ let map_binding ~config ~empty_loc ~pstr_loc ~file_name ~rec_flag binding =
            Exp.fun_ ~arity:None nolabel None
              (Pat.var @@ Location.mknoloc "ref")
              inner_expression
-         else inner_expression)
+         else inner_expression (* Todo: filter out any attributes? *))
```

Comment:
@cristianoc would this need some filtering you think?

Reply:
I think it's OK as is.
If attributes were OK on the original function, they should be the OK after transform.